README.md

```diff
@@ -114,6 +114,8 @@ The SDL2 frontend (freej2me-sdl.jar) accepts the same command-line arguments for
 
 When running under Microsoft Windows please do note paths require an additional `/` prefixed. For example, `C:\path\to\midlet.jar` should be passed as `file:///C:\path\to\midlet.jar`
 
+Special note for Windows: It is recommended to use Adoptium's [OpenJDK JRE](https://adoptium.net/temurin/releases/?os=windows&arch=x64&package=jre), instead of Oracle JRE. Late versions of Oracle introduced a bootstrapper javaw.exe, which will leave the actual javaw.exe process behind once the game is closed in RetroArch. Unfortunately, there is no good way for the core to determine which javaw is the _actual_ one, so you will either need to edit your system env to add the actual javaw.exe's path and delete the `javapath` one, or simply use Adoptium's JRE which does not have this issue.
```

Comment:
This one might become a non-issue if FreeJ2ME manages to get built into a native image by GraalVM. Haven't had the time to try this out, much less integrate it with ant, but if all goes well, Java might not even be needed to run the core.

Just a note for later on. Notifying users about the current issues is perfectly fine.